fv3core/tests/savepoint/translate/translate_a2b_ord4.py

```diff
@@ -80,5 +80,6 @@ def compute_from_storage(self, inputs):
             nord_col,
             nord_col,
         )
+        inputs["grid_type"] = 0
```

Comment:
Does this need a _TODO_ as part of the change for grid_type to an enum?

Reply:
It probably should indeed. I'll kill that PR and open a PR from a branch I can tweak.